.ci/linux/releaseWithTimeStamp.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+#
+# This script packes the project as a release without the
+# sample projects
+#
+set -euxo pipefail
+
+# Remove sample projects - and other we don't want to pack
+dotnet sln remove test/Elastic.Apm.PerfTests/Elastic.Apm.PerfTests.csproj
+dotnet sln remove test/Elastic.Apm.AspNetFullFramework.Tests/Elastic.Apm.AspNetFullFramework.Tests.csproj
+dotnet sln remove sample/AspNetFullFrameworkSampleApp/AspNetFullFrameworkSampleApp.csproj
+dotnet pack --version-suffix alpha-`date -u +"%Y%m%d-%H%M%S"` -c Release
```

Comment:
This file is the same as `release.sh`, except we add the parameter 

```
--version-suffix alpha-`date -u +"%Y%m%d-%H%M%S"`
```

If there is a nicer way to do this (e.g. removing this file, and have only `release.sh` and add the `--version-suffix` based on an input parameter), that'd be very nice. So feel free to improve this.

Reply:
Let me tweak the script to use a parameter and apply the same for the Jenkinsfile :)

Reply:
See https://github.com/gregkalapos/apm-agent-dotnet/pull/9

Reply:
Nice, thanks @v1v.